R/constructNWISURL.r

```diff
@@ -335,7 +335,7 @@ constructWQPURL <- function(siteNumber,parameterCd,startDate,endDate,zip=FALSE){
 constructUseURL <- function(years,stateCd,countyCd,categories){ 
 
     if(is.null(stateCd)){
-      baseURL <- drURL("useNat", format="rdb", rdb_compression="value")
+      baseURL <- drURL("useNat", format="rdb", rdb_compression="value", Access=pkg.env$access)
```

Comment:
seems fine, just wondering if in the future this `Access` param can be added in a single spot for all NWIS calls (maybe right before the `GET`) unless it needs to be added in all spots. Not an issue worth addressing though either way, just a note.